var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -52,16 +52,12 @@ class Openblas(Package):
     patch('make.patch')
 
     @property
-    def blas_libs(self):
+    def libs(self):
         shared = True if '+shared' in self.spec else False
         return find_libraries(
             ['libopenblas'], root=self.prefix, shared=shared, recurse=True
         )
 
-    @property
-    def lapack_libs(self):
```

Comment:
@davydden To add more on that : `openblas` is fairly well-behaved, so with this implementation you can even remove `Openblas.libs` and have the code still working. The default handler will do just the right thing...